rl_equation_solver/utilities/loss.py

```diff
@@ -0,0 +1,18 @@
+"""Collection of loss functions"""
+import torch
+
+
+class LossMixin:
+    """Mixin class with collection of loss functions"""
```

Comment:
Mixin -- what does this mean?

Reply:
General term for classes that aren't meant to be super classes just supposed to "mix in" a collection of methods.